shapely/_geometry_helpers.pyx

```diff
@@ -1,3 +1,4 @@
+# cython: freethreading_compatible=True
```

Comment:
I think we should also be able to specify that in setup.py for all cython files?

Reply:
Yes, good call. I have added that directive to the Cythonize call.